src/sql/parts/modelComponents/modelEditor/modelViewInput.ts

```diff
@@ -46,14 +53,30 @@ export class ModelViewInput extends EditorInput {
 	}
 
 	public get container(): HTMLElement {
-		if (!this._container && !this._dialogPane) {
-			this._container = DOM.$('div.model-view-container');
-			this._dialogPane = new DialogPane(this.title, this.modelViewId, () => undefined, this._instantiationService);
-			this._dialogPane.createBody(this._container);
-		}
 		return this._container;
 	}
 
+	public appendViewModelContainer(): void {
+		if (!this._dialogPane) {
+			this.createDialogPane();
+		}
+		if (!this._container.contains(this._dialogPaneContainer)) {
+			this._container.appendChild(this._dialogPaneContainer);
+		}
+	}
+
+	public removeModelContainer(): void {
```

Comment:
Minor: a few inconsistent names... `removeModelContainer` here, `appendViewModelContainer` above, and `hideOrRemoveModelViewContainer` in the editor